SwiftMethodSignatures.md

```diff
@@ -0,0 +1,16 @@
+In swift we can declare methods in different ways, like;
+
+```swift
+fileprivate func configureCommentCell(_ cell: PostCommentCell, atIndexPath indexPath: IndexPath) {
+	...
+}
+```                  
+
+```swift
+fileprivate func configure(_ cell: PostCommentCell, at indexPath: IndexPath) {
+	...
+}
+```                  
+
+For now there is no strict rule to pick one of two. However, for consistency we recommend to use the approach which is used in an existing project.  
```

Comment:
This is useful, but I think we should add some bad examples as well. For instance I saw something like this the other day: `configureComment(cell: PostCommentCell, ...`

Reply:
added